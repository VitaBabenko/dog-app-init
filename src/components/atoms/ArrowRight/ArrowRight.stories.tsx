import type { Meta, StoryObj } from '@storybook/react';
import { ArrowRight } from '.';

const meta = {
  title: 'Atoms/ArrowRight',
  component: ArrowRight,
  argTypes: {
    state: {
      options: ['default'],
      control: 'radio'
    }
  }
} satisfies Meta<typeof ArrowRight>;

export default meta;
type Story = StoryObj<typeof meta>;

export const Primary: Story = {
  args: {
    state: 'default'
  }
};
