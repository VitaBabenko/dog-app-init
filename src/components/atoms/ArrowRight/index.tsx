import { SvgIcon } from '@mui/material';

import RightArrow from '../../../assets/icons/RightArrow.svg?react';

export const ArrowRight = () => (
  <SvgIcon component={RightArrow} inheritViewBox />
);
