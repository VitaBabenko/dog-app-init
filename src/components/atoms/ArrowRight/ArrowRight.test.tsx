import { composeStories } from '@storybook/react';
import * as stories from './ArrowRight.stories';
import { testSnapshot } from '../../../test/utils/snapshot.utils';

describe('ArrowRight', () => {
  const { Primary } = composeStories(stories);
  testSnapshot(<Primary />);
});
