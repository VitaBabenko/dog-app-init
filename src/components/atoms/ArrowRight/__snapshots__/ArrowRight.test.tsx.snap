// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArrowRight should match snapshot Component 1`] = `
<div>
  <test-file-stub
    aria-hidden="true"
    classname="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-1v8usww-MuiSvgIcon-root"
    focusable="false"
  />
</div>
`;
